.gitlab-ci.yml

```diff
@@ -20,14 +20,21 @@
       - '.vendor'
       - 'vendor'
 
-.setup_bundler_env: &setup_bundler_env
+.setup_env: &setup_env
   before_script:
     - '(find .vendor | wc -l) || :'
     - bundle || gem install bundler --no-rdoc --no-ri
     - rm -f Gemfile.lock
     - rm -rf pkg/
     - bundle install --no-binstubs --jobs $(nproc) --path=.vendor "${FLAGS[@]}"
 
+.setup_env_beaker: &setup_env_beaker
```

Comment:
`gem install`ing on our gitlab runners causes 90% of the failures, so it shouldn't be ran. There is already a bundler on the system

Reply:
That won't work the way (it sounds like) you'd expect.  There are legitimate cases where the individual jobs will need to run `gem install` and `bundle`.   Ensuring that bundler is installed on the runner's system won't address that, and it's impossible to predict new gems that might be required.   Some jobs might need specific versions of bundler, too.  

It would be better to address why the runners are failing on `gem install` (and, probably, to stop running `gem install` and `bundle` when they aren't needed).

* Discussion at https://github.com/simp/pupmod-simp-aide/pull/34#discussion_r173586418 
* Issue at [SIMP-4520](https://simp-project.atlassian.net/browse/SIMP-4520), 
* A recent simp-ssh commit included a [`.gitlab-ci.yml` with an enhanced `&setup_bundler_env`][1] that isn't  failing to `gem install` on the runners (which turned out―among other things―to include a runner with an unwritable gem path).

[1]: https://github.com/simp/pupmod-simp-ssh/commit/30f0a47f011d850e16679a28266fb6e39143c565#diff-1

Reply:
I've updated `.gitlab-ci.yml` to run the enhanced `&setup_bundler_env`.  It's succeeding cleanly at https://gitlab.com/chris.tessmer/pupmod-simp-pki/pipelines/18924350 .